charts/airflow/README.md

```diff
@@ -1556,6 +1556,33 @@ Parameter | Description | Default
 <hr>
 </details>
 
+### `triggerer.*`
```

Comment:
_(not related to this section)_

Can you please also add the triggerer to the ["How to use Pod affinity, nodeSelector, and tolerations?"](https://github.com/airflow-helm/charts/blob/df99a33e3278de9f6935e747e8317794966ed979/charts/airflow/README.md?plain=1#L1374-L1378) section of the FAQ:

```yaml
## airflow triggerer
triggerer:
  nodeSelector: {}
  affinity: {}
  tolerations: []
```